hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/dispatcher/MonoTransactionDispatcher.java

```diff
@@ -227,9 +236,9 @@ private void dispatchPrng(@NonNull final HandleContext handleContext) {
     private void finishUtilPrng(@NonNull final HandleContext handleContext) {
         final var recordBuilder = handleContext.recordBuilder(SingleTransactionRecordBuilder.class);
         final var entropy = recordBuilder.entropy();
-        if (entropy.kind() == EntropyOneOfType.PRNG_NUMBER) {
+        if (entropy.kind() == TransactionRecord.EntropyOneOfType.PRNG_NUMBER) {
             sideEffectsTracker.trackRandomNumber((Integer) entropy.value());
-        } else if (entropy.kind() == EntropyOneOfType.PRNG_BYTES) {
+        } else if (entropy.kind() == TransactionRecord.EntropyOneOfType.PRNG_BYTES) {
             sideEffectsTracker.trackRandomBytes(PbjConverter.asBytes((Bytes) entropy.value()));
```

Comment:
Why are these lines modified?

Reply:
I did not change them manually.  It could be one of these two - 
1. Because I touched that file spotless formatted it 
2. I changed IntelliJ setting to not use * imports

Reply:
Spotless has done this to me too in the past